docs/source/contributing/documentation.rst

```diff
@@ -318,6 +318,24 @@ elements that are either a ``str`` or ``None``;
 ``(str, int)``; ``Tuple[:class:`int`, ...]`` for a tuple of variable
 length with only integers.
 
+.. note::
+   As a helper for tool for typesets, you can use ``typestring-parser`` which can 
```

Comment:
can you like to `typestring-parser`?

Reply:
the pip website link for it or the github page?